kim_tools/ase/core.py

```
@@ -933,262 +937,1284 @@ def energy_cheat(a, dimer, large_cell_len, offset, einf):
-def fcc_atoms_in_supercell(n_cells_per_side: int) -> int:
+        try:
+            if calc is not None and hasattr(calc, "clean"):
+                calc.clean()
+        except Exception:
+            pass
+
+        try:
+            if calc is not None and hasattr(calc, "__del__"):
+                calc.__del__()
+        except Exception:
+            pass
```

Ruff (0.16.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)